src/js/components/Button/StyledButton.js

```diff
@@ -153,13 +153,11 @@ const StyledButton = styled.button`
     props.focus && (!props.plain || props.focusIndicator) && focusStyle}
   ${props =>
     !props.plain &&
+    props.theme.button.transition &&
     `
-    transition-property: color,
-      background-color,
-      border-color,
-      box-shadow;
-    transition-duration: 0.1s;
-    transition-timing-function: ease-in-out;
+    transition-property: ${props.theme.button.transition.properties.join(',')};
```

Comment:
Is it possible to add space after the comma? just to the snapshot changes will be reverted to their default, and for the text to be more readable.

Reply:
I tried this. But adding a space to the join() doesn't seem to have any effect on the output. Do you have a clue why this might happen?

Reply:
Not at the top of my head, have you checked this https://stackoverflow.com/questions/5080165/join-an-array-by-a-comma-and-a-space

Reply:
The code I'm using is:
```
transition-property: ${props.theme.button.transition.properties.join(', ')};
```
Means they should be separated by a comma and a space. Still styled-components doesn't seem to respect this, and trims the spaces in the generated CSS.

The same happens when I try this in a sandbox: https://codesandbox.io/s/hopeful-herschel-bzdk7. The logged output is separated by spaces, yet the contents of the style tag added by styled-components aren't. Weird!

Apparently I cannot control the generated CSS when joining the properties like this. Since this is about formatting in the generated snapshots, are you sure this is worth the effort?

Reply:
I'm OK with how it is now.